main.nf

```diff
@@ -103,13 +123,11 @@ if( workflow.profile == 'awsbatch') {
 
 // Validate pipeline variables
 // These variable have to be defined in the profile configuration which is referenced in nextflow.config
-if (!params.genome) {
-    exit 1, "Mandatory parameter genome not specified!"    
-}
-
 if (params.fasta) {
     fasta = file(params.fasta)
     if(!fasta.exists()) exit 1, "Fasta file not found: ${params.fasta}"
+} else {
+    if (!params.genome) exit 1, "Genome parameter not specified!"
```

Comment:
This makes it sound like `--genome` wasn't specified, when really you're testing for `--fasta`.

```suggestion
    if (!params.genome) exit 1, "Fasta reference not specified!"
```

Reply:
Either you provide fasta or genome. I don't like `else if` statements.

Reply:
Ah sorry, I read the diff wrong. Ignore me.

Reply:
👍

Reply:
Updated to: `You have to specify either fasta file or Genome version!`